app/Http/Controllers/Bible/BiblesController.php

```diff
@@ -194,7 +196,10 @@ public function index()
                 BibleTransformer::class,
                 new DataArraySerializer()
             );
-            return $bibles_return->paginateWith(new IlluminatePaginatorAdapter($bibles));
+            $result =  $bibles_return->paginateWith(new IlluminatePaginatorAdapter($bibles));
+            Log::error('Size of the response body in Bytes:' . strlen(json_encode($result)));
```

Comment:
I think it should be 
returnvalue = bibles_first()
Log.error ('size of response...')
return returnvalue;